security/oidc-client-mutual-tls/src/test/java/io/quarkus/ts/security/oidcclient/mtls/Pkcs12OidcMtlsIT.java

```diff
@@ -44,4 +48,24 @@ protected KeycloakService getKeycloakService() {
     protected String getKeystoreFileExtension() {
         return P12_KEYSTORE_FILE_EXTENSION;
     }
+
+    private static boolean cannotRunOnFIPS() throws IOException, InterruptedException {
+        List<String> OsReleaseCommandOutputLines = new ArrayList<>();
+        new Command("grep", "^NAME", "/etc/os-release").outputToLines(OsReleaseCommandOutputLines).runAndWait();
+        boolean isUbuntuLinux = OsReleaseCommandOutputLines.get(0).matches(".*[Uu]buntu.*");
+
+        if (!isUbuntuLinux) {
+
+            String javaVersion = new PropertyLookup("java.version").get();
+            String javaVMVendor = new PropertyLookup("java.vm.vendor").get();
+
+            List<String> commandOutputLines = new ArrayList<>();
+            new Command("sysctl", "crypto.fips_enabled").outputToLines(commandOutputLines).runAndWait();
+            boolean isFipsEnabled = commandOutputLines.get(0).matches(".*1");
+
+            return javaVersion.matches("11.*") && javaVMVendor.matches(".*Red.*Hat.*") && isFipsEnabled;
+        }
+
+        return true;
```

Comment:
To sum this method up:

- Ubuntu => don't run
- (OpenJDK Red Hat 11) && FIPS enabled => don't run

Questions:

- Why shouldn't we run `Pkcs12OidcMtlsIT` on Ubuntu?
- We check OS for `sysctl crypto.fips_enabled` command returns error. I know exceptions are expensive, but dividing world on Ubuntu / the rest feels strange. You could avoid OS system check by catching error. BUT OS check is acceptable when done properly. What with happen when `sysctl cryptofips_enabled` is run on Windows? Because right now `Pkcs12OidcMtlsIT` is disabled on Github CI Windows, but in a week or 2, we will run it on our  own Windows with Docker, what then? Then even `grep` feels problematic. Can you please adjust this check to work on all OS types? I think it can be as simple as `(RH 11 && RHEL && FIPS enabled)` (in that exact order, OS only evaluated when 11 RH)